### ISSUE TITLE
Visual Glitch when Starting a Video

### ISSUE THREAD

**Issue by MetalNeo** [FreeTubeApp/FreeTube#785]
**Behavior of the program**
When a video beings to play, occasionally it will experience a graphical issue where the main video player is resized over and over while trying to initially load: https://gfycat.com/giddyunfitdowitcher. If I wait long enough, the player will go back to normal.

**Expected behavior**
I expect the video to remain the same size while it is loading.

**To Reproduce**
Please add all steps to reproduce the behavior:
1. Click a video to start playing it.
2. The video begins to resize itself constantly when initializing.

**Screenshots**
[-1605369875884.log](https://github.com/FreeTubeApp/FreeTube/files/5541052/-1605369875884.log)
[-1605369908300.log](https://github.com/FreeTubeApp/FreeTube/files/5541055/-1605369908300.log)
There were more logs, but the logs went blank and became unresponsive (crash?) after I saved the second log entry.

**Environment Information (please complete the following information):**
 - OS: Arch Linux
 - OS Version: Rolling
 - FreeTube version: FreeTube Git (commit 1349)
 - Installation Method (If applicable): AUR package

**Additional context**
I just deleted all of the files under dashFiles when testing another issue, but this issue was occurring before I performed this action.

**Comment by GilgusMaximus** [comment]
Hi. So that bug is very weird.
I do not have this bug, even on the video in the provided GIF. Maybe it is platform specific to Arch / Linux, as I am on Windows.

How often do you encounter this issue?

So any help would be appreciated

**Comment by MetalNeo** [comment]
I would estimate I experience this issue once every 10-30 videos played. It doesn't happen very consistently. I have been working around it by using the previous and next arrows in the top left to refresh the page. This usually fixes the issue.

**Comment by MetalNeo** [comment]
I experienced the issue again. This time with [this video](https://invidious.snopyta.org/watch?v=FW2GHNtupvE). I encountered #786 when trying to copy the exact logs. All that I could get was a screenshot of the error messages that kept looping 
while the video was resizing itself over and over.

![error](https://user-images.githubusercontent.com/53447663/99880517-b81b6e80-2c0b-11eb-8c7f-75174fdc7ef9.png)

This occurred on the portable version of nightly 310.

**Comment by GilgusMaximus** [comment]
So from what I can see, the original error is the connection refused error, which then leads to all other errors. Although I cannot see why the connection was refused. Additionally the video grabbing from the URL is part of a module we use, so usually not in our domain.

Do you have some kind of network devices that filter traffic from certain domains or something similar, which might block the Google domains?

**Comment by MetalNeo** [comment]
I use [blahdns](https://blahdns.com) as my primary DNS server. It is meant to block ads and malware domains, but it may be blocking the requests to certain Google domains occasionally. I'll do some testing to see if I can verify if there is a domain that is being blocked by it.

**Comment by MetalNeo** [comment]
I am closing this issue since it seemed to stem from DNS server conflicts on my end and not the main application.

**Comment by PrestonN** [comment]
To answer your last question, yes the URL you mentioned is essential to play videos. The only way to bypass this requirement would be to proxy the videos through Invidious.

Glad you were able to resolve the issue though.
